packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.tsx

```diff
@@ -15,12 +15,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-/* @flow strict */
 
-import PropTypes from 'prop-types';
 import React from 'react';
+// @ts-expect-error Untyped import. Not added as a stub declaration so consumers still have the suppression in their node_modules when shipped untranspiled.
 import { cssModules } from 'bpk-react-utils';
+// @ts-expect-error Untyped import. Not added as a stub declaration so consumers still have the suppression in their node_modules when shipped untranspiled.
 import BpkText from 'bpk-component-text';
+// @ts-expect-error Untyped import. Not added as a stub declaration so consumers still have the suppression in their node_modules when shipped untranspiled.
```

Comment:
Tested prior to this addition with `@skyscanner/backpack-web@9.2.0-beta.1`.

This proved:

We need to update the script that transforms from multi to single package to also target imports in TS files. This was addressed in https://github.com/Skyscanner/backpack/commit/2d795915283b71ad3aafa7fc7ce25eb15e220570.

It also showed that `npm start` via `react-scripts` typecheck runs also against non-transpiled node_modules.

When running with `"noImplicitAny": true` this causes a problem, as these imports error.

To fix this (in the interim before they are converted to TS themselves) that means either:

1. Consumers need to add lines like `declare module '*/bpk-component-text';` to their stub declaration files to account for the `../../bpk-component-text` import it will be transformed to
2. We can add ts-expect-error inline in each case. This is more ugly inside Backpack, but less work for consumers. The advantage of ts-expect-error over ts-ignore is that when the suppression is no longer needed it will generate an error, reminding us to clear it up.

I've gone for (2) here, feeling it lower effort for consumers while we incrementally convert all of Backpack over.

WDYT?

Reply:
cc: @kirstybryce

Reply:
Rather than a long inline comment, we could also have a shortened inline comment that references a decision doc?

Reply:
I think the context may be worth keeping around past this PR, so have put it in /decisions and updated so that each suppression directs people there. This should help people to understand why, and also if someone finds a better way make it easier for them to propose it.

Reply:
Tested in `@skyscanner/backpack-web@9.2.0-beta.3` with ts-expect-error

Reply:
and cc: @LexSwed

Reply:
I love `// @ts-expect-error` with decision record link 😍